app/components/team/Index.js

```diff
@@ -127,13 +128,11 @@ class TeamIndex extends React.PureComponent {
             <Panel.Header>
               Frequently Asked Team Questions
             </Panel.Header>
-            <Panel.Section className="max-width-2">
-              <h3 className="mt3 h4 bold">How do teams work with SSO?</h3>
-              <p>When a user signs in to SSO, the additional user is added to your account, and will be charged immediately, just as if you had invited them to the account.</p>
-              <h3 className="mt3 h4 bold">A question about teams?</h3>
-              <p>An answer about teams.</p>
-              <h3 className="mt3 h4 bold">A question about teams?</h3>
-              <p>An answer about teams.</p>
+            <Panel.Section className="max-width-3">
+              <h3 className="mt3 h4 bold">How do I disable Teams?</h3>
+              <p>To disable teams you need to delete all your teams, and then select “Disable Teams”</p>
+              <h3 className="mt3 h4 bold">How does Teams work with SSO?</h3>
+              <p>When a user joins the organization via SSO, they’ll be automatically added to any teams that have the “Automatically add new users to this team” setting enabled.</p>
```

Comment:
How about these for some other questions:

- Can I manage teams via the API?
- Will enabling Teams break any of my integrations?
- Will all my organisation members loose access to pipelines when I enable Teams?
- Does it cost any extra?